llvm/test/CodeGen/AMDGPU/convergence-laneops.ll

```diff
@@ -0,0 +1,65 @@
+; RUN: llc -stop-after=amdgpu-isel -mtriple=amdgcn-- -mcpu=gfx1100 -verify-machineinstrs -o - %s | FileCheck --check-prefixes=CHECK,ISEL %s
+
+; CHECK-LABEL: name:            basic_readfirstlane_i64
+;       CHECK:    [[TOKEN:%[0-9]+]]{{[^ ]*}} = CONVERGENCECTRL_ANCHOR
```

Comment:
We don't need these isel checks for the tokens if the IR verifier is going to catch the breakage. This test can jus be merged with the existing intrinsic tests

Reply:
I currently see machine verifier failure which is not related to this patch. An  i32 example with trunc here, https://godbolt.org/z/he8asMe77. 
This is also seen with wider type legalizations that we do now, so I cannot integrate these with existing tests just yet. am I missing something here ?

Reply:
I thought you switched to using the intrinsics such that this would avoid the verifier error. This test is going to fail if you are still hitting the error

Reply:
this is a preexisting error, and the failure is further down the pipeline. (after sreg alloc now i guess), does it make sense to have it as xfail now rather then stopping after isel?

Reply:
My worry is this test won't get updated whenever that's fixed. How about you leave the existing checks, but add an additional "not --crash" run line that checks the verifier error?

Reply:
Makes sense, updated.